hackathon_site/dashboard/frontend/src/pages/Teams/Teams.tsx

```diff
@@ -1,5 +1,43 @@
-import React from "react";
-// import styles from "./Teams.module.scss";
+// import React from "react";
```

Comment:
please remove these comments and remove your testing popup modal. If you'd like to implement your modal you could do it for cancel order on the participant side, check line 216 (PendingTables) in ItemTable.tsx to see the cancel order button.